test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -1382,3 +1382,15 @@ func.func @torch.aten.roll(%arg0: !torch.vtensor<[?,?],f32>, %arg1: !torch.int,
   %2 = torch.aten.roll %arg0, %0, %1 : !torch.vtensor<[?,?],f32>, !torch.list<int>, !torch.list<int> -> !torch.vtensor<[?,?],f32>
   return %2 : !torch.vtensor<[?,?],f32>
 }
+
+// -----
+// CHECK-LABEL:   func.func @aten.index.Tensor_hacked_twin(
+// CHECK-SAME:                  %[[INP:.*]]: !torch.vtensor<[?],f32>, %[[INDEX:.*]]: !torch.vtensor<[?,?],si64>) -> !torch.vtensor<[?,?],f32> {
+// CHECK:           %[[INDICES:.*]] = torch.prim.ListConstruct %[[INDEX]] : (!torch.vtensor<[?,?],si64>) -> !torch.list<vtensor>
+// CHECK:           %[[RES:.*]] = torch.aten.index.Tensor %[[INP]], %[[INDICES]] : !torch.vtensor<[?],f32>, !torch.list<vtensor> -> !torch.vtensor<[?,?],f32>
+// CHECK:           return %[[RES]] : !torch.vtensor<[?,?],f32>
+func.func @aten.index.Tensor_hacked_twin(%arg0: !torch.vtensor<[?],f32>, %arg1: !torch.vtensor<[?,?],si64>) -> !torch.vtensor<[?,?],f32> {
```

Comment:
This test is no longer required. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#why-so-much-end-to-end-testing

> DON'T use a unit test if your lowering pattern could be described as a trivial "macro expansion" of one op into another op or set of ops. That is, if you feel like your unit test is just rewriting b.create<...>(...) into CHECK: ... then it is probably not a useful unit test.